### PR TITLE
Changes to ItMiiDynamicUpdatePart1 and ItMiiDynamicUpdatePart2 for v9

### PR DESCRIPTION
**Pull request by vanajamukkara**
Changes to ItMiiDynamicUpdatePart1 and ItMiiDynamicUpdatePart2 for v9. Other classes which use scaleCluster methods are also modified for just the method call.

Jenkins results - failures because of defects
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/12342/
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/12351/

Defects
https://jira.oraclecorp.com/jira/browse/OWLS-101872
https://jira.oraclecorp.com/jira/browse/OWLS-101921

### CODE REVIEW

**Review by anpanigr — APPROVED** [review]
lgtm

**Review by sankarpn — CHANGES_REQUESTED** [review]
can you resync your branch to the latest changes from cluster-reference? I see some of the code is already in cluster-reference branch.

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java [hunk 1]
```diff
@@ -211,6 +211,16 @@ private static void initializeGenericKubernetesApiClients() {
             apiClient //the api client
         );
 
+    clusterCrdClient =
```
Comment (sankarpn) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java:214]: we already have clusterResClient created

Comment (vanajamukkara) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java:214]: > can you resync your branch to the latest changes from cluster-reference? I see some of the code is already in cluster-reference branch.

I am in sync with cluster-reference.
[vmukkara@vmukkara-1 weblogic-kubernetes-operator]$ git pull origin cluster-reference
From https://github.com/oracle/weblogic-kubernetes-operator
 * branch            cluster-reference -> FETCH_HEAD
Already up-to-date.

**Review by sankarpn — APPROVED** [review]
lgtm

**Pull request merged**